README.md

```diff
@@ -1,5 +1,151 @@
+# The Bowling Challenge
 
-Bowling Challenge
+This weekend (and next) the focus is on JavaScript. This is a totally new language and concept for me. My weekend challenge will be focusing on what have I learned about JavaScript this week, and how can I continue to use the same process to learn more about it. The challenge itself this week is complex so I am going to focus on understanding the business logic first, implementing the user interface second.
+
+## Technologies used
+
+- JavaScript for most of the functionality
+- HTML and CSS for the web page
+- Jasmine for the tests
+- ESLint for the linter
+- Node.js and travis for automated tests
+- Karma for running automated tests - note this is not yet working
+    - `npm install karma --save-dev`
+    - `npm install karma-jasmine karma-firefox-launcher --save-dev`
+    - `npm install -g karma-cli`
+
+
+## Setting up the project
+- `git clone `
+- `bundle install`
+- `npm i -g eslint` to install the linter
+
+## Using the project
+
+#### To run the project
+
+- `open -a "Google Chrome" index.html` in the console
+
+#### To run tests
+
+- To run Jasmine tests, type `open -a "Google Chrome" SpecRunner.html` into the console
+
+#### To run the linter (eslint)
+
+- To run the linter, type `eslint src/**`
+
+## My approach
+
+- Setup of a basi git repo so that I can make notes and feel ready to build
+- Understand the rules of bowling and model how that all works
+- TDD a game, frame and bonus class
+- End of first weekend - basic scores and strikes/spares working but does not work for two strikes in a row
```

Comment:
good to split it more evenly between 2 weekends, and accomplished a lot in the first weekend.